src/rpc.rs

```diff
@@ -20,18 +20,42 @@ pub struct SubmitTransactionRequest {
     pub transaction: Transaction
```

Comment:
Agree that it is better to have types with a suffix `Vx` if they have new versions. Breaking change will happen but it can enforce the developers to be aware of the version of the types that they use.